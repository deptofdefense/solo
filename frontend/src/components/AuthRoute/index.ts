export { default } from "./AuthRoute";
export { default as HomeRoute } from "./HomeRoute";
