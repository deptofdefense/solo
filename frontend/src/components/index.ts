export { default as PostLogoutPage } from "./PostLogoutPage";
export { default as Navbar } from "./Navbar";
export { default as LoginPage } from "./LoginPage";
export { default as Layout } from "./Layout";
export { default as HomePage } from "./HomePage";
export { default as AuthRoute, HomeRoute } from "./AuthRoute";
export { default as StatusPage } from "./StatusPage";
export { default as Table } from "./Table";
