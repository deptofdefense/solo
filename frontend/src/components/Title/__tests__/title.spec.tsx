import React from "react";
import { Title } from "components";
import { render } from "test-utils";

describe("title component", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<Title>Test Title</Title>);
    expect(asFragment()).toMatchSnapshot();
  });

  it("Includes title being passed as children", () => {
    const { queryByText } = render(<Title>Title Argument</Title>);
    expect(queryByText("Title Argument")).toBeInTheDocument();
  });
});
