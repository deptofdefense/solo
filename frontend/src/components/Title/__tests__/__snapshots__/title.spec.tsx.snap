// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title component matches snapshot 1`] = `
<DocumentFragment>
  <h1
    class="usa-prose"
  >
    Test Title
  </h1>
</DocumentFragment>
`;
